Extensions/AzureBlob/Cosmos.DataTransfer.AzureBlobStorage/AzureBlobDataSink.cs

```diff
@@ -40,15 +41,37 @@ public async Task WriteToTargetAsync(Func<Stream, Task> writeToStream, IConfigur
 
             logger.LogInformation("Saving file '{File}' to Azure Blob Container '{ContainerName}'", settings.BlobName, settings.ContainerName);
 
+            var lastLogTime = DateTime.UtcNow;
+            var logInterval = TimeSpan.FromMinutes(1);
+            long totalBytes = 0;
+
             await using var blobStream = await blob.OpenWriteAsync(true, new BlockBlobOpenWriteOptions
             {
                 BufferSize = settings.MaxBlockSizeinKB * 1024L,
                 ProgressHandler = new Progress<long>(l =>
                 {
-                    logger.LogInformation("Transferred {UploadedBytes} bytes to Azure Blob", l);
+                    if (DateTime.UtcNow - lastLogTime >= logInterval)
+                    {
+                        logger.LogInformation("{BlobName}: transferred {TotalMiB} MiB to Azure Blob", settings.BlobName, (double) l / 1024/1024);
```

Comment:
Maybe add formatting so it's not megalong? Could format it as bytes and put dots in between if you want to know the exact bytes (N format) or MiB, KiB with x numbers after the decimal (Nx so N3 would give 3 past the dot)